### ISSUE TITLE
[Feature Request] Media 'Cleanup' function

### ISSUE THREAD

**Issue by Ltek** [syd711/vpin-studio#223]
As I've been updating my FullDMD videos, I've noticed a lot of media that no longer has an associated table. A function that scans all Popper media folders and deletes media that is not associated to a table would be a nice addition to keep the file count down.

**Comment by syd711** [comment]
Isn't this a duplicate of https://github.com/syd711/vpin-studio/issues/134 ?

This feature is under development from the Visual Pinball Back Manager guy. I'll integrate it once he is finished.

**Comment by Ltek** [comment]
coool

**Comment by syd711** [comment]
I close this as duplicate and will contact Matt (@Malibu) about the status.
